doc/source/contributing.rst

```diff
@@ -552,8 +552,8 @@ use cases and writing corresponding tests.
 Adding tests is one of the most common requests after code is pushed to *pandas*.  Therefore,
 it is worth getting in the habit of writing tests ahead of time so this is never an issue.
 
-Like many packages, *pandas* uses the `Nose testing system
-<https://nose.readthedocs.io/en/latest/index.html>`_ and the convenient
+Like many packages, *pandas* uses `pytest`
```

Comment:
the backtick at the end of pytest is incorrect, as the 'env' is closed with the backtick after the link on the following line